packages/web/components/LandingPage/Resources/Item/index.js

```diff
@@ -11,7 +11,7 @@ const Item = ({ title, image, description, link, buttonDisabled }) => {
 				<S.Image src={image.src} alt={image.alt} />
 				<S.Body>
 					<S.Description>{description}</S.Description>
-					<S.Link href={!buttonDisabled && link.href} buttonDisabled={buttonDisabled}>
+					<S.Link href={link.href} buttonDisabled={buttonDisabled}>
```

Comment:
Was throwing an error in development because href was a boolean (false)

Reply:
Awesome! 🚀